distribution/docs/src/main/jdocs/content/_endpoints/opensearch-endpoint-contents.adoc

```diff
@@ -179,12 +179,12 @@ Make sure to repeat the starting point as the last point to close the polygon.
 
 Sorting by date will sort the effective date.
 
-|`maxResults`
+|`pageSize`
```

Comment:
❓ 1. Can we change this or does changing this parameter count as a change to the public API? If so, I suggest we _add_ `pageSize` and ensure that the semantics between `pageSize` and `maxResults` are logically distinct.
❓ 2. If we're going to change (or add), then the abbreviated form should match. `ps` instead of `mr`.